wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -41,6 +41,13 @@
 <div class="well">
     {{ object.abstract.rendered|safe }}
 </div>
+{% if show_private_notes and perms.edit_private_notes and object.private_notes %}
```

Comment:
@madduck Given @drnlm's vote, let's remove the setting and `show_private_notes` and go back to always showing it if the user has the `edit_private_notes` permission?

Apologies for the back and forth on this.